src/Raven.Client/ServerWide/Operations/OneTimeBackupOperation.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Documents.Operations.Backups;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Json.Serialization;
+using Sparrow.Json;
+
+namespace Raven.Client.ServerWide.Operations
+{
+    public class OneTimeBackupOperation : IServerOperation<OperationIdResult>
```

Comment:
just `BackupOperation`
should be `IMaintenanceOperation` ?

Reply:
renamed, I followed the logic of `RestoreBackupOperation` so I put it in `IServerOperation`